roles/node_images_base/vars/packages/suse.yml

```diff
@@ -23,151 +23,118 @@
+  - crash=7.3.0-150400.3.5.8
+  # rationale: Necessary for the dynamic kernel module support.
   - dkms=2.6.1-bp154.1.30
-  - dmidecode=3.4-150400.16.8.1
-  - dump=0.4b47-150400.1.10
-  - ebtables=2.0.11-1.1
-  - elfutils-lang=0.185-150400.5.3.1
-  - elfutils=0.185-150400.5.3.1
-  - emacs=27.2-150400.3.6.1
+  #
+  ## Admin experience / sanity
+  #
+  # rationale: Desired for easy navigation through bourne again shell (BASH).
+  - bash-completion=2.7-150400.13.3.1
+  # rationale: Desired for easy viewing of kernel modules.
+  - kmod-bash-completion=29-4.15.1
+  # rationale: Desired for easy viewing of files.
+  - less=590-150400.3.3.1
+  # rationale: Desired for easy navigation of NVME devices in bourne again shell (BASH).
+  - nvme-cli-bash-completion=2.0+30.g86f82c58cb97-150400.3.15.1
+  # rationale: Desired for easy navigation of NVME devices in Zshell.
+  - nvme-cli-zsh-completion=2.0+30.g86f82c58cb97-150400.3.15.1
+  # Analysis / Triage data Collection
+  # rationale: Desired for viewing and dumping Ethernet controller and port information.
   - ethtool=5.14-150400.1.6
-  - expect=5.45.4-3.3.1
-  - fontconfig=2.13.1-150400.1.4
-  - fonts-config=20200609+git0.42e2b1b-4.7.1
-  - fping=4.0-4.3.2
-  - fuse=2.9.7-3.3.1
-  - gdb=12.1-150400.15.6.1
-  - genders=1.27.3-150400.8.7
-  - git-core=2.35.3-150300.10.27.1
-  - gnuplot=5.4.3-150400.1.6
-  - hdparm=9.62-150400.1.7
-  - helm=3.11.2-150000.1.21.1
-  - helm-bash-completion=3.11.2-150000.1.21.1
-  - helm-zsh-completion=3.11.2-150000.1.21.1
-  - hpe-yq=4.33.3-1
-  - htop=3.0.5-bp154.1.30
-  - hwloc=2.5.0-150400.1.9
+  # rationale: Desired for easy usage of ip commands.
   - iproute2-bash-completion=5.14-150400.1.8
+  # rationale: Desired for basic IPv4 and IPv6 probing (provides ping).
   - iputils=20211215-150400.3.3.2
-  - issue-generator=1.7-1.17
-  - kmod-bash-completion=29-4.15.1
-  - ledmon=0.94-150400.5.5
-  - libasm1=0.185-150400.5.3.1
-  - libbd_mdraid2=2.26-150400.1.5
-  - libcanberra-gtk0=0.30-150400.13.10
-  - libdw1=0.185-150400.5.3.1
-  - libelf-devel=0.185-150400.5.3.1
-  - libelf1=0.185-150400.5.3.1
-  - libfreebl3-hmac=3.79.4-150400.3.29.1
-  - libfreebl3=3.79.4-150400.3.29.1
-  - liblldp_clif1=1.1+44.0f781b4162d3-3.3.1
-  - libopenssl1_1=1.1.1l-150400.7.42.1
-  - libpwquality1=1.4.4-150400.15.4
-  - libstoragemgmt=1.8.5-3.3.1
-  - libsystemd0=249.16-150400.8.25.7
-  - libudev1=249.16-150400.8.25.7
-  - libunwind-devel=1.5.0-4.5.1
+  # rationale: Desired for viewing and dumping small computer system interface (SCSI) devices.
   - lsscsi=0.28-1.24
+  # rationale: Necessary for viewing manuals.
   - man=2.7.6-6.22
-  - nvme-cli-bash-completion=2.0+30.g86f82c58cb97-150400.3.15.1
-  - nvme-cli-zsh-completion=2.0+30.g86f82c58cb97-150400.3.15.1
-  - open-lldp=1.1+44.0f781b4162d3-3.3.1
+  # rationale: Necessary for openscap.
   - openscap-utils=1.3.6-150400.11.3.1
+  # rationale: Necessary for running CVE scans.
   - openscap=1.3.6-150400.11.3.1
-  - openssh-clients=8.4p1-150300.3.18.2
```

Comment:
I see that these are removed but don't exist anywhere else in the PR? I remember that we added this because an upgrade or something broke the build, they might not even need to be pinned anymore?

Reply:
`openssh` requires `openssh-clients` and `openssh-server`, installing `openssh` will install both.